docs/content/operations/tls-support.md

```diff
@@ -31,6 +31,16 @@ values for the below mentioned configs among others provided by Java implementat
 |`druid.server.https.certAlias`|Alias of TLS/SSL certificate for the connector.|none|yes|
 |`druid.server.https.keyStorePassword`|The [Password Provider](../operations/password-provider.html) or String password for the Key Store.|none|yes|
 
+The following table contains configuration options related to client certificate authentication.
+
+|Property|Description|Default|Required|
+|--------|-----------|-------|--------|
+|`druid.server.https.requireClientAuth`|If set to true, clients must identify themselves by providing a TLS certificate.  If `requireClientAuth` is false, the rest of the options in this table are ignored.|false|no|
+|`druid.server.https.trustStoreType`|The type of the trust store containing certificates used to validate client certificates.|`java.security.KeyStore.getDefaultType()`|no|
```

Comment:
This won't be used for anything other than verifying client certificates?

Reply:
yes, the truststore on the server configuration side is only used to check the client certificates